app/controllers/blog_controller.rb

```diff
@@ -5,7 +5,7 @@ class BlogController < ApplicationController
   layout "layouts/blog/index"
 
   def index
-    @front_matter = { "title" => "Blog" }
+    @front_matter = { "title" => "Get Into Teaching Blog" }
```

Comment:
Haven't tried it, but shouldn't it work if we just symbolise this key and keep the rest as they are? :thinking:

Reply:
I think we use strings everywhere except here. It's unintuitive though, and easy to miss as `@front_matter[:something]` will return nil.

Reply:
Makes sense to change the symbols to strings then.